### ISSUE TITLE
Image missing alt text

### ISSUE THREAD

**Issue by huguestennier** [jackmoore/zoom#134]
`aria-role="presentation"` was added in #121, but having no alt text results in invalid HTML.

Also, [empty alt text](https://www.w3.org/TR/WCAG20-TECHS/H67.html) is the recommended way for ATs to ignore that image.

**Comment by jackmoore** [comment]
I believe that's correct and easy enough to fix. Thank you.
